src/System.Private.CoreLib/src/System/StubHelpers.cs

```diff
@@ -1453,42 +1453,83 @@ internal struct NativeVariant
 #endif
     }  // struct NativeVariant
 
+    internal abstract class CleanupWorkListElement
+    {
+        private CleanupWorkListElement m_Next;
+        protected abstract void DestroyCore();
+
+        public void Destroy()
+        {
+            DestroyCore();
+            var next = m_Next;
```

Comment:
This should have explicit type (`CleanupWorkListElement `) not `var` according to the coding convention.